WARN  2014-04-21 18:34:10,519 (NullSarosSession.java:64) unexpected method call
WARN  2014-04-21 18:34:10,519 (NullSarosSession.java:64) unexpected method call
WARN  2014-04-21 18:34:15,065 (NullSarosSession.java:64) unexpected method call
WARN  2014-04-21 18:34:15,074 (NullSarosSession.java:64) unexpected method call
WARN  2014-04-21 18:34:27,588 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-04-21 18:34:48,240 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-21 18:35:18,267 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
ERROR 2014-04-21 22:05:06,586 (SarosNet.java:95) XMPP connection error: 
java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
ERROR 2014-04-21 22:05:06,586 (SarosNet.java:385)  new connection state == ERROR
java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
WARN  2014-04-22 08:24:09,972 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-04-22 08:24:09,972 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.91.162:0
ERROR 2014-04-22 08:24:20,723 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-22 08:24:38,101 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-04-22 08:24:38,101 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.91.162:0
ERROR 2014-04-22 08:24:48,932 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-22 08:25:33,411 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.91.162:0
WARN  2014-04-22 08:25:33,411 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
ERROR 2014-04-22 08:25:44,129 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-22 08:26:20,888 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-04-22 08:26:20,888 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.91.162:0
ERROR 2014-04-22 08:26:31,595 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-22 08:26:41,954 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.91.162:0
WARN  2014-04-22 08:26:41,954 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-04-22 08:26:45,126 (NullSarosSession.java:64) unexpected method call
WARN  2014-04-22 08:26:45,126 (NullSarosSession.java:64) unexpected method call
ERROR 2014-04-22 08:26:52,817 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-22 08:26:52,817 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@10c72d61, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:273)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-22 08:26:52,817 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@7c48faa4, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:275)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-04-22 08:26:52,817 (SWTUtils.java:193) could not execute runnable de.fu_berlin.inf.dpp.ui.util.SWTUtils$1@785ab9ba, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTSync(SWTUtils.java:194)
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSWTSync(SWTUtils.java:157)
	at de.fu_berlin.inf.dpp.ui.util.DialogUtils.popUpYesNoQuestion(DialogUtils.java:242)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:675)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
ERROR 2014-04-22 08:26:52,817 (DialogUtils.java:250) An internal error ocurred while trying to open the question dialog.
WARN  2014-04-22 08:26:53,021 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@11b83070, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:248)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:681)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
